src/main/java/org/circle8/dao/PuntoReciclajeDao.java

```diff
@@ -37,13 +34,23 @@ public class PuntoReciclajeDao extends Dao{
 		    WHERE 1=1
 		""";
 
+	private static final String SELECT_GET = """
+		   SELECT pr."ID", pr."Titulo", pr."Latitud", pr."Longitud", pr."DiasAbierto", pr."CiudadanoId" , prtr."TipoResiduoId", tr."Nombre", ciu."UsuarioId"
+		   	FROM "PuntoReciclaje" AS pr
+		   		LEFT JOIN "PuntoReciclaje_TipoResiduo" AS prtr ON prtr."PuntoReciclajeId" = pr."ID"
+				LEFT JOIN "TipoResiduo" AS tr on tr."ID" = prtr."TipoResiduoId"
+				LEFT JOIN "Ciudadano" AS ciu on pr."CiudadanoId" = ciu."ID"
+		    WHERE pr."ID" = ?
+		    	AND pr."CiudadanoId" = ?
```

Comment:
Hay una combinacion de tabs y espacios que me esta rompiendo los ojos.
```suggestion
		SELECT pr."ID", pr."Titulo", pr."Latitud", pr."Longitud", pr."DiasAbierto", pr."CiudadanoId" , prtr."TipoResiduoId", tr."Nombre", ciu."UsuarioId"
		  FROM "PuntoReciclaje" AS pr
		  LEFT JOIN "PuntoReciclaje_TipoResiduo" AS prtr ON prtr."PuntoReciclajeId" = pr."ID"
		  LEFT JOIN "TipoResiduo" AS tr on tr."ID" = prtr."TipoResiduoId"
		  LEFT JOIN "Ciudadano" AS ciu on pr."CiudadanoId" = ciu."ID"
		 WHERE pr."ID" = ?
		   AND pr."CiudadanoId" = ?
```